draft-ysl-cats-metric-definition.md

```diff
@@ -94,67 +90,72 @@ This document uses the following terms defined in {{!I-D.ietf-cats-framework}}:
 
 ## Level 0: Raw Metrics
 
 Level 0 metrics encompass detailed, raw metrics, including but not limit to:
 
-- CPU: Base Frequency, Number of Cores,  Boosted Frequency, Memory Bandwidth, Memory Size, Utilization Ratio, Core Utilization Ratio, Power Consumption.
+- CPU: Clock Frequency, number of cores, boosted frequency, memory bandwidth, memory size, utilization ratio, core utilization ratio, Power Consumption.
 - GPU: Frequency, Number of Render Unit, Memory Bandwidth, Memory Size, Memory Utilization Ratio, Core Utilization Ratio, Power Consumption.
 - NPU: Computing Power, Utlization Ratio, Power Consumption.
 - Network: Bandwidth, Capacity, Throughput, TXBytes, RXBytes, HostBusUtilization.
 - Storage: Available Space, Read Speed, Write Speed.
 - Delay: Time taken to process a request.
 
-Detailed information of a metric in L0 can be encoded into Application Programming Interface(API)(e.g., Restful API), and different services have their own metrics with different information elements. L0 metrics are used widely in IT systems.
+L0 metrics can be encoded into an Application Programming Interface (API), such as a RESTful API, and can be solution-specific. Different resources can have their own metrics, each conveying unique information about their status. These metrics can generally have units, such as bits per second (bps) or floating point instructions per second (flops).
 
-Regarding network related raw metrics, IPPM WG has defined many types of metrics in {{performance-metrics}}. {{?RFC9439}} also defines many metrics of packet performance and Throughput/Bandwidth. Regarding computing metrics, {{?I-D.rcr-opsawg-operational-compute-metrics}} lists a set of cloud resource metrics.
+Regarding network-related information, the IPPM WG has defined various types of metrics in {{performance-metrics}}. Additionally, in {{?RFC9439}}, the ALTO WG has introduced an extended set of metrics related to packet performance and throughput/bandwidth. For compute metrics, {{?I-D.rcr-opsawg-operational-compute-metrics}} lists a set of cloud resource metrics.
 
 ## Level 1: Normalized Metrics in Categories
 
-The metrics in L1 are categorized into different categories, and abstraction will be applied to each category. L0 raw metrics can be classified into multiple categories, such as computing, networking, storage, and delay. In each category, the metrics are normalized into a value that present the state of a resource. Potential categories are shown below:
+L1 metrics are organized into distinct categories, such as computing, networking, storage, and delay. Each L0 metric is classified into one of these categories. Within each category, a single L1 metric is computed using an *aggregation function* and normalized to a unitless score that represents the performance of a resource according to that category. Potential categories include:
```

Comment:
```suggestion
L1 metrics are organized into distinct categories, such as computing, networking, storage, and delay. Each L0 metric is classified into one of these categories. Within each category, a single L1 metric is computed using an *aggregation function* and normalized to a unitless score that represents the performance of resources according to that category. Potential categories include:
```
a resource or resources?

Reply:
Sounds good. I would like to further suggest using "the underlying resources". I will make that change and update the PR. Thanks.